src/plugins/console/server/lib/proxy_request.test.ts

```diff
@@ -115,39 +116,33 @@ describe(`Console's send request`, () => {
       } as any;
     });
 
-    it('should decode percent-encoded uri pathname and encode it correctly', async () => {
-      const uri = new URL(
-        `http://noone.nowhere.none/%{[@metadata][beat]}-%{[@metadata][version]}-2020.08.23`
-      );
-      const result = await proxyRequest({
-        agent: null as any,
-        headers: {},
-        method: 'get',
-        payload: null as any,
-        timeout: 30000,
+    const runEncodePathTests = async (initialPath: string, expectedPath: string, uri?: URL) => {
```

Comment:
[nit] I think this util could be used regardless if the path was encoded. You might consider renaming it to something like: `verifyRequestPath()`